.github/workflows/rotki_nightly.yml

```diff
@@ -54,7 +54,7 @@ jobs:
           key: ${{ runner.os }}-testdir
       - name: Install dependencies
         run: |
-          pip install --upgrade "pip<19.0.0" wheel
+          pip install --upgrade pip wheel
```

Comment:
Hey @yabirgb looks okay but can you maybe pin a pip version? Cause our packaging may break if pip suddenly releases an incompatible version.